src/main/rules/GCI96/GCI96.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
in this PR, there are some lacks : you should put the modification of

- RULES.md to add inside this rule in the array
- update CHANGELOG.md